src/dune_console/dune_console.ml

```diff
@@ -93,9 +97,18 @@ module Backend = struct
 
     let reset () = Dumb.reset ()
 
+    let finish () = set_status_line None
```

Comment:
This is the line fixing bug.

Reply:
Could you clarify which bug this is fixing?

Reply:
The one in the changelog #6700

Reply:
Thanks!

Reply:
Can we separate the bug fix from the rest of the changes? It doesn't seem nice to couple bug fixes with logic reorganisation.

Reply:
Done in https://github.com/ocaml/dune/pull/6821